packages/common/src/components/Navbar/Navbar.tsx

```diff
@@ -61,6 +61,7 @@ const Navbar: React.FC<INavbar> = ({
       )}
       {showUserProfileCard && profileList?.length !== 0 && (
         <NavbarUserProfileCard
+          // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
I think this can be avoided by checking if `!profileList` is `true` or `false`

Reply:
Added the check [here](https://github.com/ACE-VSIT/website/pull/84/commits/8914a284da6e212cfea4f17a78882a4574511707#diff-e3ab400afd2bf48fc0c8b7d611e17e76408ed98e264ea10b057a55faab48f6ebL62-R62)